.github/workflows/test_integration.yml

```diff
@@ -45,8 +45,10 @@ jobs:
         action-operator:
           - { lxd-channel: "5.21/stable", cloud: "lxd", cloud-channel: "5.21", juju: "2.9" }
           - { lxd-channel: "5.21/stable", cloud: "lxd", cloud-channel: "5.21", juju: "3" }
+          - { lxd-channel: "latest/stable", cloud: "lxd", cloud-channel: "latest", juju: "3" }
```

Comment:
todo: move to new PR. these changes are a drive by with no documentation. It can also be dangerous to use the latest channel with lxd and juju. You can easily run into lxd/juju failures.